swift/ql/src/queries/Security/CWE-328/WeakSensitiveDataHashing.qhelp

```diff
@@ -51,12 +51,17 @@
             </li>
         </ul>
 
+        <p>
+            Note that special purpose algorithms exist for message authentication (ensuring that a message comes from a particular sender). These algorithms should be used when appropriate, as they address common vulnerabilities of simple hashing schemes in this context.
+        </p>
+
     </recommendation>
     <example>
 
         <p>
-            The following examples show a function for checking whether the hash
-            of a certificate matches a known value -- to prevent tampering.
+            The following examples show a function for fetching data from a
+            URL along with a hash of the data, perhaps to check the data has
+            not been tampered with.
 
             In the first case the MD5 hashing algorithm is used that is known to be vulnerable to collision attacks.
```

Comment:
Adding a newline so that the content that is currently on line 66 is treated as a new paragraph.
```suggestion

            In the first case the MD5 hashing algorithm is used that is known to be vulnerable to collision attacks.
```

Reply:
I think the issue was that we need a `</p>` and new `<p>` to start a new paragraph properly here.  I've pushed a fix.